commands-yml/commands/mobile-command.yml

```diff
@@ -84,6 +84,7 @@
       | mobile:acceptAlert | Accepts an on-screen alert | Optional button label to click on | <none> |
       | mobile:dismissAlert | Dismisses an on-screen alert | Optional button label to click on | <none> |
       | mobile:scrollBackTo | Scroll from one element to another | `{elementId, elementToId}` | `{elementId: 2, elementToId: 1}` |
+      | mobile:scroll | Scroll the given scrollable element until the element identifier by `strategy` and `selector` becomes visible since Appium 1.18.0. An optional `maxSwipes` property can limit the number of scroll actions. It accepts an optional `element` property which should be the element identifier and scrollable. Without this property, Appium will select the first scrollable container on the view which will be selected by default. Please read `mobileScroll` section in [appium-uiautomator2-driver](https://github.com/appium/appium-uiautomator2-driver/blob/master/lib/commands/general.js) for more details. | `{element, strategy, selector, maxSwipes}` | `{strategy: 'accessibility id', selector: 'target content description'}` |
```

Comment:
..Appium will select the first scrollable container on the view which will be selected by default.. -> Appium will select the first currently available scrollable container